### PR TITLE
LPS-195817 Filter Cached doesn't work with Fragments Provided OOTB

### PR DESCRIPTION
**Pull request by Mikellorza**
https://liferay.atlassian.net/browse/LPS-195817

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by Mikellorza** [comment]
ci:test:sf

File: modules/apps/layout/layout-reports-web/src/main/java/com/liferay/layout/reports/web/internal/struts/GetLayoutReportsLayoutItemDataStrutsAction.java [hunk 1]
```diff
@@ -210,7 +210,8 @@ httpServletResponse, new DummyWriter())),
 
 			jsonArray.put(
 				JSONUtil.put(
-					"cached", fragmentEntryLink.isCacheable()
```
Comment (ealonso) [inline modules/apps/layout/layout-reports-web/src/main/java/com/liferay/layout/reports/web/internal/struts/GetLayoutReportsLayoutItemDataStrutsAction.java:213]: Hey @Mikellorza,

The cached value should be the one in the fragmentEntryLink, if it is not present we should find why it is not there

**Comment by Mikellorza** [comment]
ci:test:sf

**Comment by Mikellorza** [comment]
ci:test:relevant

**Comment by Mikellorza** [comment]
ci:test:sf

**Comment by rafaellajordao** [comment]
The failures are not related to this PR [(comparison with acceptance routine here)](https://testray.liferay.com/home/-/testray/runs/compare?testrayRunIdB=2420185341&testrayRunIdA=2422282548&view=teams). See https://github.com/brianchandotcom/liferay-portal/pull/140564

**Comment by rafaellajordao** [comment]
ci:close

**Pull request closed without merging**